scripts/build-image.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+set -e
+CURL="curl -sSf"
+FLANNEL_CNI_ROOT=$(git rev-parse --show-toplevel)
+IMAGE_NAME=quay.io/coreos/flannel-cni
+VERSION=$($FLANNEL_CNI_ROOT/scripts/git-version)
+CNI_VERSION="v0.5.2"
```

Comment:
I didn't remove the Makefile just for local dev, but the image on jenkins only uses build-image.sh
I totally fine with removing the makefile altogether. @aaronlevy